.github/ISSUE_TEMPLATE/custom_product/tech-day.md

```diff
@@ -49,46 +65,56 @@ This checklist is focused on a classic Jahia repository (module, app)
+- [ ] 🔝 A job running Sonar checks (including OWASP Dependency Check) is executed regularly
+### QA / Automated Tests
+- [ ] 🚨 The codebase is compatible with the latest release of Jahia
 - [ ] 🔝 Automated tests are using jahia-cypress for all utils functions
 - [ ] 🔝 The test framework is using page-object models published by other modules
 - [ ] 🔝 The test framework is publishing its own page-object models for use by others
+- [ ] 🟠 A manual-run workflow is available
+- [ ] 🟠 Instructions and [test cases](https://jahia.testrail.net/index.php?/dashboard) are available to document how a release should be tested (how to do the "sanity check" of this module)
 - [ ] 🙏 Automated tests are using a recent version of Cypress
 - [ ] 🙏 Automated tests are only relying on supported modules
 ### CI/CD
 - [ ] 🔝 The build and the release workflows use the JDK 11 image (only if Jahia Parent is set to 8.2.0.0+)
-- [ ] 🔝 GitHub Actions (nightlys and other workflows) are executed without warnings (such as depreciations)
+- [ ] 🔝 GitHub Actions (nightlys and other workflows) are executed without warnings nor errors (such as depreciations, failed tests, ...)
 - [ ] 🙏 The latest version of the actions are used (including jahia-modules-action)
+- [ ] 🙏 GitHub Actions [reusable workflows](https://github.com/Jahia/jahia-modules-action/tree/main/.github/workflows) are used
 ### Documentation
+- [ ] 🔝 Readme.md is up-to-date (module purpose, technical details, configuration steps)
+- [ ] 🟠 A tech roadmap is available 
 - [ ] 🙏 Module's documentation available on the academy is up-to-date
+### Issues
+- [ ] 🟠 If the repository is public, issues/pull requests from the community have been reviewed and answered, if answer was not possible, a PM/DM was notified.
 ### GitHub
-- [ ] 🟠 [Branch protection](https://confluence.jahia.com/display/PR/GitHub+%28Product%29+-+Ref+ISPOL08.A14025#GitHub(Product)RefISPOL08.A14025-Branchprotection) is enabled for the repository
+- [ ] 🚨 [Branch protection](https://jahia-confluence.atlassian.net/wiki/spaces/PR/pages/2067858/GitHub+Product+-+Ref+ISPOL08.A14025#Branch-protection) is enabled for the repository
 - [ ] **Automatically delete head branches** is selected in **Settings**
-- [ ] 🙏 The repository contains a README.md file
 - [ ] 🙏 Repository topics match are populated (at a minimum: "product" and "supported")
 - [ ] 🙏 Stale branches or branches older than 2 years (non-maintenance branches) have been removed
 
-## Fork day checklist
+## Fork checklist
 
 This checklist is focused on our forked repositories
```

Comment:
Most of the time our fork is related to a specific version, ensure to switch to our maintenance branch before starting.